include/deal.II/lac/trilinos_precondition.h

```diff
@@ -1618,7 +1618,8 @@ namespace TrilinosWrappers
    * except for the higher_order_elements parameter which does not exist in
    * PreconditionerAMGMueLu.
    *
-   * This class requires Trilinos 11.14 or higher.
+   * This class requires Trilinos 11.14 or higher. At the moment 64bit-indices
```

Comment:
Should we make this more visible with a `@note`, and the warning about the unstable interface with a `@warning`?